recipe/meta.yaml

```diff
@@ -20,28 +24,37 @@ build:
     - fairseq-score = fairseq_cli.score:cli_main
     - fairseq-train = fairseq_cli.train:cli_main
     - fairseq-validate = fairseq_cli.validate:cli_main
-  script: {{ PYTHON }} -m pip install . -vv
-  skip: true  # [win]
+  script:
+    # compiling Cython token_block_utils_fast.pyx module so it can be imported
+    - {{ PYTHON }} setup.py build_ext --inplace
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+    # - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation --global-option="build_ext" --global-option="--inplace"
```

Comment:
why the additional step?

Reply:
it was confirmed after many attempts, that calling `{{ PYTHON }} setup.py build_ext --inplace` separately from `{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation` will cythonize the `*.pyx` files that that setup later needs. This would not be the case if we compiled the steps, even with `global-option="--inplace"`.

I suspect this is due to some misconfiguration of their build whereby the `*.pyx` files are referenced as already _cythonized_ modules (i.e. a `*.pyc` is referenced before its creation...the `*.pyc` is the result of the cythonization) as opposed to being _cythonized_ during the build step itself. We also examined the build output and confirmed the first of the two steps did the `*.pyx` --> `*.pyc` conversion.

This step is also referenced in many places such as [here](https://github.com/facebookresearch/fairseq/blob/920a548ca770fb1a951f7f4289b4d3a0c1bc226f/fairseq/data/data_utils.py#L325).

Reply:
Thanks @boldorider4 for clarifying. I've added a comment to refer to their build command.